src/_utils/conditional.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function conditional(condition, allowedConditionAction, deniedConditionAction) {
```

Comment:
Using `conditional` as the function name makes it really difficult to get it's use.
Maybe calling it  `ifCondition` using the third argument as an "else" statement, and making it default to `false`.

By doing this, if an user called this method could work like:

```
write: ifCondition(
  dataIsString,
  anyCondition(
   dataIsOne(['A', 'B', 'C])
  )
)
```

as well as:

```
write: ifCondition(
  dataIsString,
  anyCondition(
   dataIs('A'), dataIs('B'), dataIs('C')
  ),
  anyConditon(
    dataIs(1), dataIs(2), dataIs(3)
  )
)
```